docs/integrations/data-connectors/fivetran.md

```diff
@@ -6,7 +6,16 @@ title: Fivetran
 
 Enabling the Fivetran integration for Statsig will allow Statsig to push events to your Fivetran account through a webhook. This allows you to forward Statsig data to any connectors available from Fivetran.
 
-## Configuration
+## Configuring Outbound Events
 
 1. Follow the steps in the [Fivetran Webhook Setup Guide](https://fivetran.com/docs/events/webhooks/setup-guide) to create a new Webhook URL.
-2. On the Statsig [Integration page](https://console.statsig.com/integrations) enable the Fivetran Integration by pasting in the Fivetran Webhook URL and click confirm.
\ No newline at end of file
+2. On the Statsig [Integrations](https://console.statsig.com/integrations) page, enable the Fivetran integration by pasting in the Fivetran Webhook URL and click **Confirm**.
+
+## Filtering Events
+
+Once you've enabled outbound events to Fivetran, you can select which categories of Statsig events you want to export by click on the **Event Filtering** button and checking the appropriate boxes as shown below.
```

Comment:
Should we just create a generic filtering integration events page and link to that?